data/meta/tokens/1/0xf7b55C3732aD8b2c2dA7c24f30A69f55c54FB717.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "token",
+  "name": "Curve sdCRV-CRV Pool",
+  "symbol": "sdCRVCRV-f",
+  "description": "This token represents a Curve factory liquidity pool. Holders earn fees from users trading in the pool, and can also deposit the LP to Curve's gauges to earn CRV emissions. This pool contains sdCRV and CRV.",
```

Comment:
This pool contains CRV and sdCRV, a tokenized veCRV from StakeDAO.

Reply:
fixed